cypress/integration/instruments.ts

```diff
@@ -601,13 +601,63 @@ context('Instrument tests', () => {
       });
       cy.contains('Proposals');
 
+      cy.get('[data-cy="reviewer-filter"] input').should(
+        'have.value',
+        ReviewerFilter.YOU
```

Comment:
A quick minor note 
We talked during the walkthrough that the site should have headings and labels from users' perspectives, not systems. e.g. should say "My proposals" instead of "Your proposals"
This should be ReviewersFilter.ME

Reply:
Great I wasn't aware of this. But will change it.